### PR TITLE
Docs: fix broken links to permissions

### PR DESCRIPTION
**Pull request by lebdron**
### Description of the Change
Fix two broken links to permission docs.

### Benefits
No broken links in docs!

### Possible Drawbacks 
None

### CODE REVIEW

File: docs/source/develop/api/permissions.rst [hunk 1]
```diff
@@ -1172,7 +1172,7 @@ Allows getting any `transaction <../../before_start/glossary.html#transaction>`_
 .. literalinclude:: ../../../../example/python/permissions/can_get_all_txs.py
     :language: python
     :linenos:
-    :lines: 11-58
+    :lines: 12-59
```
Comment (MBoldyrev) [inline docs/source/develop/api/permissions.rst]: maybe we should add a comment to that python source that these line numbers have to be kept matching it?

File: docs/source/develop/api/permissions.rst [hunk 2]
```diff
@@ -1172,7 +1172,7 @@ Allows getting any `transaction <../../before_start/glossary.html#transaction>`_
 .. literalinclude:: ../../../../example/python/permissions/can_get_all_txs.py
```
Comment (LiraLemur) [inline docs/source/develop/api/permissions.rst]: docs/source/develop/api/permissions.rst is an autogenerated file. 
This needs to be fixed in /example/python/permissions/can_get_all_txs.py

Comment (lebdron) [reply docs/source/develop/api/permissions.rst]: I ran `make permissions` and got this file. The file that is currently in the repository is outdated.

Comment (LiraLemur) [reply docs/source/develop/api/permissions.rst]: My bad, I thought that permissions were made during the build process

File: docs/source/develop/api/permissions.rst (hunk 1 above)
Comment (lebdron) [reply docs/source/develop/api/permissions.rst]: Boundaries are defined here https://github.com/hyperledger/iroha/blob/e8dc37b4244d6332e4fb3967e54e989d7e6870d5/docs/permissions_compiler/rst.py#L111-L137

**Pull request merged**